.gitlab-ci.yml

```diff
@@ -7,13 +7,13 @@ libtelio-build-pipeline:
     LIBTELIO_PROJECT_PATH: $CI_PROJECT_PATH
     LIBTELIO_DEFAULT_BRANCH: $CI_DEFAULT_BRANCH
 
-    # Don't checkout libtelio submodule when using GIT_SUBMODULE_STRATEGY.
+    # Don't checkout libtelio submodule when using GIT_SUBMODULE_STRATEGY. 
     # This ensures that all jobs that use `libtelio` submodule are calling
     # `ci/checkout.sh`, which uses `LIBTELIO_COMMIT_SHA` to
     # checkout the correct libtelio REF.
     GIT_SUBMODULE_PATHS: ':(exclude)libtelio'
 
   trigger:
     project: $LIBTELIO_BUILD_PROJECT_PATH
-    branch: v0.4.8
+    branch: rust-platform-verifier
```

Comment:
This will need to be changed to version with https://bucket.digitalarsenal.net/low-level-hacks/vpn/client/libtelio-build/-/merge_requests/156 merged